disnake/role.py

```diff
@@ -366,6 +366,26 @@ def is_assignable(self) -> bool:
             and (me.top_role > self or me.id == self.guild.owner_id)
         )
 
+    def has_custom_gradient(self) -> bool:
+        """
+        Whether the role has custom gradient.
```

Comment:
I don't know about using the word "custom" here, it implies the existence of non-custom gradient